docs/examples/sock-shop.md

```diff
@@ -200,35 +224,45 @@ Path: ./deploy/kubernetes/manifests
 Branch: master
 Type: kustomize
 
-✚ Generating Source manifest
+◎ Checking cluster status
+✔ GitOps installed
+✚ Generating application spec manifest
 ✚ Generating GitOps automation manifests
-✚ Generating Application spec manifest
-► Cloning ssh://git@github.com/example/microservices-demo.git
-► Writing manifests to disk
-► Applying manifests to the cluster
+► Adding application "microservices-demo" to cluster "kind-kind" and repository
 ► Committing and pushing gitops updates for application
 ► Pushing app changes to repository
 >
 ```
 
-After running this, not only is the *Sock Shop* running in the cluster, but we can observe and modify the GitOps resources for the application. Here's how they look in the application repository:
+So, it's just like the example above except we didn't have to call out the location of the configuration repository. Regardless of whether or not the application manifests are stored in the configuration repository, though, the configuration itself is stored in a special directory (`.weave-gitops`) at the top level of the configuration repository:
```

Comment:
didn't have to explicitly call out

Reply:
@sympatheticmoose Can you clarify what the feedback is here?